diff --git a/lib/main.dart b/lib/main.dart
--- a/lib/main.dart
+++ b/lib/main.dart
@@ -93,6 +93,7 @@ class ArchiveApp extends StatelessWidget {
       splashFactory: NoSplash.splashFactory,
       splashColor: Colors.transparent,
       highlightColor: Colors.transparent,
+      shadowColor: Colors.transparent,
       colorScheme: ColorScheme.fromSeed(
         seedColor: _button,
         surface: _bg,
